### PR TITLE
Patch imperial queues

### PR DESCRIPTION
**Pull request by apeltzer**
---
name: New Config
about: A new cluster config
---

Please follow these steps before submitting your PR:

- [ ] If your PR is a work in progress, include `[WIP]` in its title
- [ ] Your PR targets the `master` branch
- [ ] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [ ] Add your custom config file to the `conf/` directory
- [ ] Add your documentation file to the `docs/` directory
- [ ] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [ ] Add your custom profile to the `README.md` file in the top-level directory
- [ ] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

### CODE REVIEW

File: conf/imperial.config [hunk 1]
```diff
@@ -37,39 +37,39 @@ profiles {
...
+                queue  = 'throughput72'
                 cpus   = { 2	 * task.attempt }
                 memory = { 48.GB * task.attempt }
                 time   = { 8 * task.attempt  }
             }
 
             withLabel:process_medium {
                 // TARGET QUEUE: throughput
-                queue = 'v1_throughput72'
+                queue = 'vthroughput72'
```
Comment (SPPearce) [inline conf/imperial.config:48]: Should this have a v at the start?

**Comment by jfy133** [comment]
@apeltzer any updates to this?

**Comment by combiz** [comment]
FYI I am no longer at Imperial full time. The HPC job name/size changes occur with a regular cadence, perhaps bi-annually, so the config would benefit from an active NF user at Imperial in the contact details. Perhaps Dr Fancy (@nfancy) can advise?

File: conf/imperial.config (hunk 1 above)
Comment (nfancy) [reply conf/imperial.config:48]: Imperial HPC has recently moved away from specifying queue names. So, we need to remove all the queue names. 

In this following conf:

```
withLabel:process_low {
                queue  = 'throughput72'
                cpus   = { 2	 * task.attempt }
                memory = { 48.GB * task.attempt }
                time   = { 8 * task.attempt  }
            }
 ```
 
 The time should be  **8.h**

**Review by nfancy — CHANGES_REQUESTED** [review]
Please remove the queue names. Add the time unit in the process_low conf. Thanks a lot.

**Comment by jfy133** [comment]
> Please remove the queue names. Add the time unit in the process_low conf. Thanks a lot.

Thanks @nfancy ! Can we also add you as the listed contact if other Imperial users have issues/questions?

**Comment by nfancy** [comment]
> > Please remove the queue names. Add the time unit in the process_low conf. Thanks a lot.
> 
> Thanks @nfancy ! Can we also add you as the listed contact if other Imperial users have issues/questions?

Yes, I am happy to be added as a contact. Nurun

**Comment by jfy133** [comment]
@nfancy I've removed all queues, fixed the unit suffix, and added you as contaxt. If this looks OK please approve and we can merge :)

**Review by nfancy — APPROVED** [review]

**Comment by nfancy** [comment]
> @nfancy I've removed all queues, fixed the unit suffix, and added you as contaxt. If this looks OK please approve and we can merge :)

Approved just now. Thank you.

**Pull request merged**